UMN-AutoPackager/public/Set-Config.ps1

```diff
@@ -0,0 +1,86 @@
+function Set-Config {
+    <#
+    .SYNOPSIS
+        Sets a value for the global configuration JSON file for the auto packager.
+    .DESCRIPTION
+        This command sets in a json file a value for one of various global configurations used as part of the AutoPackager. It requires a the full path and name of the JSON file, the key to set, and the value to set it to.
+    .EXAMPLE
+        Set-Config -Path .\config.json -Key CompanyName  -Value "University of Minnesota"
+        Sets the value of CompanyName to University of Minnesota for the config.json file
+    .PARAMETER Path
+        The full path and file name of the JSON file to be updated
+    .PARAMETER Key
+        The key that you want to update in the Global Configuation JSON file
+    .PARAMETER Value
+        The value you want to set the key to
+    #>
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$True, HelpMessage = "The full path and file name of the JSON file to be updated")]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path,
+        [Parameter(Mandatory=$True, HelpMessage = "The key that you want to update in the Global Configuation JSON file")]
+        [ValidateNotNullOrEmpty()]
+        [string]$Key,
+        [Parameter(Mandatory=$True, HelpMessage = "The value you want to set the key to")]
+        [ValidateNotNullOrEmpty()]
+        [string]$Value
+    )
+    begin {
+        Write-Verbose -Message "Starting $($myinvocation.mycommand) Updating $key with $value"
+    }
+    process {
+        $json = Get-Content $Path -Raw
+        $config = ConvertFrom-Json -InputObject $json
```

Comment:
Maybe use the new Get-Config function here to make things more dynamic and future-proof?

Reply:
Added the function get-UMNGlobalconfig.

Reply:
Great! Resolving.